js/ui/editor/editor.d.ts

```diff
@@ -62,6 +63,12 @@ export interface EditorOptions<TComponent> extends WidgetOptions<TComponent> {
      * @public
      */
     validationMessageMode?: ValidationMessageMode;
+    /**
+     * @docid
+     * @default "bottom"
+     * @public
+     */
+     validationMessagePosition?: ValidationMessagePosition;
```

Comment:
"auto" value type should be published only in components where it's useful. It it you useful for slider or CheckBox?

Reply:
done